packages/@azure/servicebus/data-plane/test/testUtils.ts

```diff
@@ -193,19 +193,26 @@ export function getReceiverClient(
 }
 
 export async function purge(
-  recieverClient: QueueClient | SubscriptionClient,
-  useSessions?: boolean
+  receiverClient: QueueClient | SubscriptionClient,
+  useSessions?: boolean,
+  sessionId?: string
```

Comment:
Instead of adding a new parameter, you can replace the existing one.
We dont need both `useSessions` and `sessionId` for `purge`.

Reply:
Will need both as each time we want to clear just one/relevant entities.
In general the purging is already unnecessary for every test if the tests clean after themselves.

Also, updated session tests to invoke purging of second session entity only when necessary which is fortunately within same describe suite in the session tests spec

Reply:
This should be fixed with https://github.com/Azure/azure-sdk-for-js/pull/1049/commits/1079ffab3cca79bada044483e76974cfa8dd0fa1